scripts/Packages/Package_NoLead__DFN_QFN_LGA_SON/size_definitions/dfn.yaml

```diff
@@ -76,6 +76,42 @@ DFN-6-1EP_2x1.8mm_P0.5mm_EP1.2x1.6mm:
   #include_suffix_in_3dpath: 'False'
   #include_pad_size: 'fp_name_only' # 'both' | 'none' (default)
 
+DFN-6-1EP_2x2mm_P0.5mm_EP0.6x1.37mm:
+  device_type: 'DFN'
+  #manufacturer: 'man'
+  #part_number: 'mpn'
+  size_source: 'https://www.analog.com/media/en/package-pcb-resources/package/pkg_pdf/ltc-legacy-dfn/05081703_C_DC6.pdf'
+  # ipc_class: 'qfn' # 'qfn_pull_back'
+  #ipc_density: 'least' #overwrite global value for this device.
+  body_size_x:
+    nominal: 2
+    minimum: 1.9
+    maximum: 2.1
+  body_size_y:
+    nominal: 2
+    minimum: 1.9
+    maximum: 2.1
+  lead_width_min: 0.2
+  lead_width_max: 0.3
+  lead_len_min: 0.3
+  lead_len_max: 0.5
+
+  EP_size_x: 0.6
+  EP_size_y: 1.37
+  # EP_paste_coverage: 0.65
+  EP_num_paste_pads: [2, 2]
```

Comment:
I suggest using only one paste pad in x-direction because the EP is so narrow.

Reply:
Done.